==81239== 72 (24 direct, 48 indirect) bytes in 1 blocks are definitely lost in loss record 31 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x1471DC: ____C_A_T_C_H____T_E_S_T____6() (arvore.cpp:67)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 72 (24 direct, 48 indirect) bytes in 1 blocks are definitely lost in loss record 32 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x14756B: ____C_A_T_C_H____T_E_S_T____8() (arvore.cpp:78)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 72 (24 direct, 48 indirect) bytes in 1 blocks are definitely lost in loss record 33 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x1478FA: ____C_A_T_C_H____T_E_S_T____10() (arvore.cpp:89)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 72 (24 direct, 48 indirect) bytes in 1 blocks are definitely lost in loss record 34 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x147C89: ____C_A_T_C_H____T_E_S_T____12() (arvore.cpp:100)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 72 (24 direct, 48 indirect) bytes in 1 blocks are definitely lost in loss record 35 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x1482BE: ____C_A_T_C_H____T_E_S_T____16() (arvore.cpp:120)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 168 (24 direct, 144 indirect) bytes in 1 blocks are definitely lost in loss record 36 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x1486AF: ____C_A_T_C_H____T_E_S_T____18() (arvore.cpp:132)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 168 (24 direct, 144 indirect) bytes in 1 blocks are definitely lost in loss record 37 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x148E67: ____C_A_T_C_H____T_E_S_T____22() (arvore.cpp:154)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
==81239== 216 (24 direct, 192 indirect) bytes in 1 blocks are definitely lost in loss record 38 of 38
==81239==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==81239==    by 0x186D49: Abb<int>* abb_insere<int>(Abb<int>*, int const&) (abb.hpp:25)
==81239==    by 0x1739CD: Abb<int>* abb_inicia<int>(std::__cxx11::list<int, std::allocator<int> > const&) (abb.hpp:17)
==81239==    by 0x148A92: ____C_A_T_C_H____T_E_S_T____20() (arvore.cpp:143)
==81239==    by 0x133B73: Catch::TestInvokerAsFunction::invoke() const (catch.hpp:14257)
==81239==    by 0x132DB2: Catch::TestCase::invoke() const (catch.hpp:14100)
==81239==    by 0x12D113: Catch::RunContext::invokeActiveTestCase() (catch.hpp:12959)
==81239==    by 0x12CE46: Catch::RunContext::runCurrentTest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) (catch.hpp:12932)
==81239==    by 0x12B8BC: Catch::RunContext::runTest(Catch::TestCase const&) (catch.hpp:12693)
==81239==    by 0x12EA95: Catch::(anonymous namespace)::TestGroup::execute() (catch.hpp:13287)
==81239==    by 0x12FEA8: Catch::Session::runInternal() (catch.hpp:13493)
==81239==    by 0x12FB9E: Catch::Session::run() (catch.hpp:13449)
==81239== 
